fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/observer/FedoraEventImpl.java

```diff
@@ -199,6 +199,20 @@ public String getEventID() {
         return eventID;
     }
 
+    /**
+     * Extract a Map with any additional information from the underlying JCR Event.
+     * @return a Map of additional information.
+     */
+    @Override
+    @SuppressWarnings("unchecked")
```

Comment:
I like to use a var inside the method to hold suppressions so that they aren't visible at the method (which, to me, is an inexact semantic), but not everyone does. Do we have a policy on this?